### PR TITLE
Edu 5490 scaffolding

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1367

Creates an empty form for the 5490.

### CODE REVIEW

File: src/js/edu-benefits/5490/helpers.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```
Comment (jbalboni) [inline src/js/edu-benefits/5490/helpers.jsx]: Can we delete everything but the transform in here? If we need to reuse any of the other data, it should be in a file that's shared between the forms.

File: src/js/edu-benefits/5490/components/ServicePeriodView.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (jbalboni) [inline src/js/edu-benefits/5490/components/ServicePeriodView.jsx]: If this is the same as the 1990, we'll probably want to share it. So maybe remove it for now.

**Review by jbalboni — APPROVED** [review]
LGTM. Made a couple comments.

File: src/js/edu-benefits/5490/helpers.jsx (hunk 1 above)
Comment (cvalarida) [inline src/js/edu-benefits/5490/helpers.jsx]: Ah, yes. I didn't even think about these. Thanks!

**Pull request merged**